homework/homework2.playground/Contents.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+let a = 2.5
+let b = Int(a)
+print(b)
+let c = a - Double(b)
+print(c)
+
+let a1 = 9.7
+let b1 = Int(a1)
+print(b1)
+let c1 = Decimal(a1) - Decimal(b1)
```

Comment:
если через Float or Double ,тогда же получается приблизительное число (например 0.40000003)

Reply:
Yeah, it's a known problem with the proximity of the fractional numbers here. It only matters for that case when you want to show this number to a user and it would display everything the way you need (you can set the necessary formatting). And when you perform some calculations it doesn't matter if it's 0.4 or 0.40000003, it won't affect them. So you can ignore this problem here

Reply:
ooooo
okey